doc/NPCs.md

```diff
@@ -492,6 +492,9 @@ Effect | Description
 `u_mod_fatigue: fatigue_int`<br/>`npc_mod_fatigue: fatigue_int` | Your character or the NPC will have `fatigue_int` added or subtracted from its fatigue.
 `u_make_sound, npc_make_sound: message_string`, `volume: volume_int`, `type: type_string`,  | A sound of description `message_string` will be made at your character or the NPC's location of volume `volume_int` and type `type_string`. Possible types are: background, weather, music, movement, speech, electronic_speech, activity, destructive_activity, alarm, combat,    alert, or order
 `u_mod_healthy, npc_mod_healthy : amount_int, cap: cap_int` | Your character or the NPC will have `amount_int` added or subtracted from its health value, but not beyond `cap_int`.
+`u_add_morale: morale_string`, (*optional* `bonus: bonus_int` ), (*optional* `max_bonus: max_bonus_int` ), (*optional* `duration: duration_int`), (*optional* `decay_start` : `decay_int`), (*optional* `capped`: `capped_bool`)<br/> `npc_add_morale: morale_string`, (*optional* `bonus: bonus_int` ), (*optional* `max_bonus: max_bonus_int` ), (*optional* `duration: duration_int`), (*optional*`decay_start` : `decay_int`), (*optional* `capped`: `capped_bool`)| Your character of the NPC will gain a morale bonus of type `morale_string`. The Morale bonus will be for 1 or `bonus_int` amount up to 1 or `max_bonus_int`. It will last for `duration: duration_int` seconds or 1 hour. It will begin to decay after `decay_int` seconds or a half hour. `capped_bool` Whether this morale is capped or not, defaults to false.
```

Comment:
```suggestion
`u_add_morale: morale_string`, (*optional* `bonus: bonus_int` ), (*optional* `max_bonus: max_bonus_int` ), (*optional* `duration: duration_int`), (*optional* `decay_start` : `decay_int`), (*optional* `capped`: `capped_bool`)<br/> `npc_add_morale: morale_string`, (*optional* `bonus: bonus_int` ), (*optional* `max_bonus: max_bonus_int` ), (*optional* `duration: duration_int`), (*optional*`decay_start` : `decay_int`), (*optional* `capped`: `capped_bool`)| Your character or the NPC will gain a morale bonus of type `morale_string`. Morale is changed by `bonus_int` (default 1), with a maximum of up to `max_bonus_int` (default 1). It will last for `duration: duration_int` seconds (default 1 hour). It will begin to decay after `decay_int` seconds (default 0.5 hours). `capped_bool` Whether this morale is capped or not, defaults to false.
```
Shouldn't the default for `max_bonus_int` be `bonus_int`, not 1 (particularly if `bonus_int` is negative!)? (And if morale isn't capped, is `max_bonus_int` effective?)

Reply:
The defaults are the default values for the parameters of the morale add function on morale.h line 33.